lib/src/components/NxTab/NxTab.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import classnames from 'classnames';
+
+import { ActiveTabContext } from '../NxTabs/NxTabs';
+
+import { Props, propTypes } from './types';
+export { Props } from './types';
+
+import './NxTab.scss';
+
+const NxTab = function NxTabElement(props: Props) {
+  // Use React.useContext instead of importing useContext for jest to mock the value in the test
+  const {activeTab, onTabSelect} = React.useContext(ActiveTabContext);
+  const {id, index, tabIndex = 0, className, onClick, onKeyPress, ...attrs} = props;
+  const active = activeTab === index;
+  const classNames = classnames('nx-tab', className, { active });
+  const selected = active ? 'true' : 'false';
+
+  function handleKeyPress(event: React.KeyboardEvent<HTMLLIElement>) {
+    if (onKeyPress) {
+      onKeyPress(event);
+    }
+    if (!event.isDefaultPrevented() && (event.key === ' ' || event.key === 'Enter')) {
```

Comment:
Why the check for `isDefaultPrevented`?

Reply:
If the user provided their own onKeyPress and called prevent default then I'd expect the tab not to be selected.